torch/ao/quantization/_dbr/auto_trace.py

```diff
@@ -325,6 +323,12 @@ def _patched_module_call(self, *args, **kwargs):
                 global_op_idx[0] = 0
 
                 output = super().__call__(*new_args, **new_kwargs)
+
+                if first_call:
+                    for _, v in self.named_modules():
+                        if hasattr(v, '_auto_quant_state'):
+                            v._auto_quant_state.insert_observers(v)
```

Comment:
Just so I understand, this is functionally the same as before, where we inserted the observers in `_first_call_op_prepare_before_hook_create_subgraphs`, because here we're calling `insert_observers` after executing the op in L325?

Reply:
Correct.  Here is a diagram of before and after:

```
# op graph
x0 -> op0 -> op1 -> x1

# before:
1. trace through op0, insert observers for op0
2. trace through op1, insert observers for op1

# after:
1. trace through op0 and op1
2. insert observers for op0 and op1
```